src/Commands/GetEntryPoints.ts

```diff
@@ -70,20 +75,18 @@ export class GetEntryPoints implements CommandModule {
         status: args.status as string[]
       });
 
-      if (args.verbose) {
-        // eslint-disable-next-line no-console
-        console.log('%j', entryPoints);
-      } else {
-        // eslint-disable-next-line no-console
-        console.log(
-          '%j',
-          entryPoints.map((entryPoint) => ({
+      const ep = args.verbose
+        ? entryPoints
+        : entryPoints.map((entryPoint) => ({
             id: entryPoint.id,
             method: entryPoint.method,
             url: entryPoint.url
-          }))
-        );
-      }
+          }));
+
+      // eslint-disable-next-line no-console
+      console.log(
+        args.pretty ? JSON.stringify(ep, null, 4) : JSON.stringify(ep)
```

Comment:
Why not use 2 instead of 4? It's more compact (this may be crucial for small terminal windows), yet still convenient and easy to read.

Reply:
sure, 2 is also ok